lighthouse-core/test/gather/gatherers/offline.js

```diff
@@ -55,21 +55,4 @@ describe('HTTP Redirect gatherer', () => {
       assert.deepEqual(offlineGather.artifact, {offlineResponseCode: -1});
     });
   });
-
-  it('handles driver evaluateAsync() failure', () => {
```

Comment:
I feel a little guilty removing a test case and not adding in anything new, but we don't call `evaluateAsync()` anymore.